src/txkube/test/test_network.py

```diff
@@ -34,7 +34,9 @@
 from twisted.internet.ssl import (
     CertificateOptions, DN, KeyPair, trustRootFromCertificates,
 )
-from twisted.internet.interfaces import IReactorSSL
+from twisted.internet.interfaces import (
+    IReactorSSL, IReactorTCP, IReactorTime, IReactorPluggableNameResolver
```

Comment:
Can you add a trailing comma to this import list?

Reply:
Fixed